MovieQuiz/Models/MostPopularMovies.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  MostPopularMovies.swift
+//  MovieQuiz
+//
+//  Created by Baev on 12/9/22.
+//
+
+import Foundation
+
+struct MostPopularMovies: Codable {
+    let errorMessage: String
+    let items: [MostPopularMovie]
+}
+
+struct MostPopularMovie: Codable {
+    let title: String
+    let rating: String
+    let imageURL: URL
+    private enum CodingKeys: String, CodingKey {
+    case title = "fullTitle"
+    case rating = "imDbRating"
+    case imageURL = "image"
```

Comment:
Обычно кейсы имеют еще один вложенный таб. Проверяйте себя с помощью форматирования кода: выделить файл и зажать `control + I`
```suggestion
    private enum CodingKeys: String, CodingKey {
        case title = "fullTitle"
        case rating = "imDbRating"
        case imageURL = "image"
```

Reply:
+